plugins/ingest-attachment/build.gradle

```diff
@@ -15,10 +15,10 @@ esplugin {
 }
 
 versions << [
-  'tika'  : '1.24',
-  'pdfbox': '2.0.19',
+  'tika'  : '1.27',
+  'pdfbox': '2.0.24',
   'poi'   : '4.1.2',
-  'mime4j': '0.8.3'
+  'mime4j': '0.8.5'
```

Comment:
I think this should be version `0.8.4`? 

This version triggers: 

```
java.lang.NoSuchMethodError: java.nio.ByteBuffer.flip()Ljava/nio/ByteBuffer;	
		at __randomizedtesting.SeedInfo.seed([92EBE05E8999BE7A:FBF685D8DB444DF8]:0)	
		at org.apache.james.mime4j.io.TextInputStream.<init>(TextInputStream.java:49)	
		at org.apache.james.mime4j.io.InputStreams.createAscii(InputStreams.java:70)	
		at org.apache.james.mime4j.codec.DecoderUtil.decodeQuotedPrintable(DecoderUtil.java:51)	
		at org.apache.james.mime4j.codec.DecoderUtil.decodeQ(DecoderUtil.java:126)	
		at org.apache.james.mime4j.codec.DecoderUtil.tryDecodeEncodedWord(DecoderUtil.java:258)	
		at org.apache.james.mime4j.codec.DecoderUtil.decodeEncodedWords(DecoderUtil.java:212)	
		at org.apache.james.mime4j.codec.DecoderUtil.decodeEncodedWords(DecoderUtil.java:145)	
		at org.apache.tika.parser.microsoft.OutlookExtractor.decodeHeader(OutlookExtractor.java:563)	
		at org.apache.tika.parser.microsoft.OutlookExtractor.normalizeHeaders(OutlookExtractor.java:529)	
		at org.apache.tika.parser.microsoft.OutlookExtractor.parse(OutlookExtractor.java:161)	
		at org.apache.tika.parser.microsoft.OfficeParser.parse(OfficeParser.java:199)	
		at org.apache.tika.parser.microsoft.OfficeParser.parse(OfficeParser.java:131)	
		at org.apache.tika.parser.CompositeParser.parse(CompositeParser.java:281)	
		at org.apache.tika.parser.AutoDetectParser.parse(AutoDetectParser.java:143)	
		at org.apache.tika.Tika.parseToString(Tika.java:568)
```

Reply:
Thanks, @martijnvg, that fixed it.